parsers/objc.c

```diff
@@ -23,17 +23,6 @@
 #include "selectors.h"
 #include "vstring.h"
 
-/* To get rid of unused parameter warning in
- * -Wextra */
-#ifdef UNUSED
-#elif defined(__GNUC__)
-# define UNUSED(x) UNUSED_ ## x __attribute__((unused))
-#elif defined(__LCLINT__)
-# define UNUSED(x) /*@unused@*/ x
-#else
-# define UNUSED(x) x
-#endif
```

Comment:
no, its API is `UNUSED(x)`, not `UNUSED`.  So yeah, in most cases here it's no a problem because you apparently can parenthesize argument definitions, but it still leads to a different (unexpected) expansion.

about `__LCLINT__` I don't know.